packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -212,18 +229,7 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
           })}
         >
           <Input readOnly onClick={showCalendarGrid} value={valueFormatter(selectedDate)} />
-          <Popup
-            open={open}
-            onOpenChange={(e, { open }) => setOpen(open)}
-            content={
-              <DatepickerCalendar
-                {...calendarOptions}
-                onDateChange={handleChange}
-                localizedStrings={DEFAULT_LOCALIZED_STRINGS}
-              />
-            }
-            trapFocus
-          >
+          <Popup open={open} onOpenChange={(e, { open }) => setOpen(open)} content={calendarElement} trapFocus>
```

Comment:
Should we be creating the components instead of using it explicit? `Popup.create`...?

Reply:
I don't have any preferences here, @layershifter what would you recommend here?

Reply:
`createShorthand` is preferred